howitworks.txt

```diff
@@ -13,22 +13,22 @@ icon:edit[] link:https://github.com/libremesh/lime-web/edit/master/howitworks.tx
   - Layer 2 roaming inside certain areas
   - Smart gateway selection with redundancy and possibility of user-choice
   - Compatibility of most of the existing scenarios
-  - A single firmware for all the network (for ground level, for backbone, for companies, for quick deployments, etc.)
+  - A single firmware image for all of your network (for ground level, for backbone, for companies, for quick deployments, etc.)
 
 == The Basics
 
-The network architecture designed for LibreMesh is based in two layers:
+The network architecture designed for LibreMesh is based on two layers:
 
 === Cloud layer 2
 
 The cloud layer 2 uses the dynamic routing protocol BATMAN-ADV
 
 image::img/bat-adv.png[batmanadv,100,float="left"]
 *******************
-B.A.T.M.A.N. advanced is a mesh routing protocol which runs in kernel space.
-Even if the network topology is made by multiple nodes and multiple hops, abstracts it to a single layer 2 collision domain.
-So from the user perspective the entire mesh will look like a single LAN.
-This architecture is very interesting for roaming purposes, thus TCP and UDP connections are not lost even when moving and changing AP.
+B.A.T.M.A.N. Advanced is a mesh routing protocol that runs in kernel space.
+Even if the network topology is made of multiple nodes and multiple hops, B.A.T.M.A.N. Advanced abstracts it to a single layer 2 collision domain.
```

Comment:
In this sentence 'collision domain' must be changed to 'broadcast domain'

Reply:
Done, thanx.